src/main/java/tak/httpHandlers/AddSeekHandler.java

```diff
@@ -0,0 +1,43 @@
+package tak.httpHandlers;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import com.sun.net.httpserver.HttpExchange;
+import com.sun.net.httpserver.HttpHandler;
+
+import tak.Seek;
+
+public class AddSeekHandler implements HttpHandler {
+    @Override
+    public void handle(HttpExchange t) throws IOException {
+        String response;
```

Comment:
still looks wip but just to note, will want to accept JSON input as well rather than a string.

Reply:
Actually looking at the implementation since the tournament knows who the players are that are facing off probably should be defined. if it's unclaimed would any user be able to join? could mess things up...

Reply:
Absolutely, I'll leave this thread unresolved until I've added that.

Reply:
The `SeekDto` has `String` fields `creator` and `opponent` with the corresponding names. When the SeekDto is sent to the seek API, it searches for currently connected players with the name in `creator`. If one is found that is not currently playing a game, the seek is created in that players name.

Otherwise the seek is not created at all and an error is returned.